parastell/magnet_coils.py

```diff
@@ -268,11 +270,30 @@ def export_step(self, step_filename="magnet_set", export_dir=""):
             self.step_filename
         ).with_suffix(".step")
 
-        coil_set = cq.Compound.makeCompound(
-            [coil.solid for coil in self.magnet_coils]
-        )
+        coil_set = cq.Compound.makeCompound(self.coil_solids)
         cq.exporters.export(coil_set, str(export_path))
 
+    def extract_solids_and_mat_tag(self, solids, mat_tags):
```

Comment:
In general, I find it a little unsettling to have functions that change the input parameters and pass them back as new output parameters.

I wonder if it's better to just return these solids and tags, and append/extend outside of the function? I guess that can be accomplished by passing in empty lists...??

If we are appending in this function, maybe the function name should be `append_solids_and_mat_tags` rather than `extract_solids_and_mat_tag`?

Reply:
Fair point! I think I prefer the former. In the case that we append/extend outside the function, it shouldn't be necessary to provide any input arguments.